src/seedsigner/gui/screens/tools_screens.py

```diff
@@ -171,6 +171,73 @@ def update_title(self) -> bool:
 
 
 
+@dataclass
+class ToolsCoinEntropyEntryScreen(KeyboardScreen):
+    def __post_init__(self):
+        # Override values set by the parent class
+        # TRANSLATOR_NOTE: Inserts the number of flips required
+        self.title = _("Coin Flip 1/{}").format(self.return_after_n_chars)
+
+        # Specify the keys in the keyboard
+        self.rows = 2
+        self.cols = 2
+        self.key_height = GUIConstants.get_top_nav_title_font_size() + GUIConstants.EDGE_PADDING/4 + 2*GUIConstants.EDGE_PADDING
+        self.keys_charset = "".join([
+            "H",
+            "T",
+        ])
+
+        # Map Key display chars to actual output values
+        self.keys_to_values = {
+            "H": "1",
+            "T": "0",
+        }
+
+        # Now initialize the parent class
+        super().__post_init__()
+
+        self.components.append(TextArea(
+            text=_("(H)eads: 1   /   (T)ails: 0"),
+            screen_y = self.keyboard.rect[3] + 3*GUIConstants.COMPONENT_PADDING,
+        ))
+
+    def update_title(self) -> bool:
+        self.title = _("Coin Flip {}/{}").format(self.cursor_position + 1, self.return_after_n_chars )
```

Comment:
Always provide a `# TRANSLATOR_NOTE` to explain the expected values for any variable insertions.

And when there's more than one variable, it's probably better to use a named variable:
```python
self.title = _(f"Coin Flip {cur_flip_num}/{num_total_flips}")
```